fail2ban/server/utils.py

```diff
@@ -116,7 +117,22 @@ def setFBlockMode(fhandle, value):
 		return flags
 
 	@staticmethod
-	def executeCmd(realCmd, timeout=60, shell=True, output=False, tout_kill_tree=True, success_codes=(0,)):
+	def buildShellCmd(realCmd, varsDict):
```

Comment:
I guess I am yet to comprehend fully... btw -- what happens when a value has a space in it or some other "interesting character"?